postgres-native-tls/src/lib.rs

```diff
@@ -180,3 +181,10 @@ where
         }
     }
 }
+
+/// Set ALPN for `TlsConnectorBuilder`
+///
+/// This is required when using `sslnegotiation=direct`
+pub fn set_postgresql_alpn(builder: &mut TlsConnectorBuilder) -> &mut TlsConnectorBuilder {
```

Comment:
It's only required for direct ssl mode. 

Actually the method is just to provide our user a shortcut for setting ALPN. Because TlsConnectionBuilder is out of our library scope, but ALPN is only writable at this builder level.

Reply:
This should return `()` imo.